app/Resources/translations/AppBundleAdmin.fr.yml

```diff
@@ -0,0 +1,2 @@
+index:
+    list_of_bluray: 'Gestion des bluray'
```

Comment:
avoid_long_sentence_for_variables :smile:
IMO (In My Opinion :wink:), bluray_list will be more appropriate.